schema/dataKeys.json

```diff
@@ -49,8 +57,7 @@
         },
         "productName": {
           "type": "string",
-          "description": "The product name e.g. exampleproduct ",
-          "pattern": "^[a-z]+$"
```

Comment:
/workspace/data/5c52c6c8-898a-4a85-93b2-9336e6601e52/data.json::$.fbc.productName: 'testProductName' does not match '^[a-z]+$'

Reply:
Or maybe just relax it to allow capital letters? Or modify e2e to contain lowercase in the name? That should be easy. Maybe just this here? https://github.com/konflux-ci/e2e-tests/blob/fd26460f5d0035b0bffe699513c558be8588b209/tests/release/pipelines/fbc_release.go#L347

Reply:
I hit this with some of the advisory keys too. I am fine with either removing it (the pattern, not the description) or modifying like Martin said so long as it is not too restricting such that users are going to fail on this

Reply:
Ok so I have allowed for the pattern to have capital letters

Reply:
looks good to me

Reply:
We also get this one `.fbc.productName: 'preGA-product' does not match '^[a-zA-Z]+$.'` I could add in to allow hyphen or just update e2e.

Reply:
Up to you. I don't know that we have the exact requirements on this field which is why I say we can just remove it, but no strong opinion